src/fabric/ScreenNativeComponent.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow strict-local
+ * @format
+ */
+/* eslint-disable */
+import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
+import type { ViewProps } from 'react-native/Libraries/Components/View/ViewPropTypes';
+import type { HostComponent } from 'react-native/Libraries/Renderer/shims/ReactNativeTypes';
+import type {
+  BubblingEventHandler,
+  Int32,
+} from 'react-native/Libraries/Types/CodegenTypes';
+
+type ScreenEvent = $ReadOnly<{||}>;
+
+type ScreenDismissedEvent = $ReadOnly<{|
+  dismissCount: Int32,
+|}>;
+
+export type NativeProps = $ReadOnly<{|
+  ...ViewProps,
+  onAppear?: ?BubblingEventHandler<ScreenEvent>,
```

Comment:
Shouldn't these events be something like `direct` instead of `bubbling`? We want them to be handled only by the `Screen` component.

Reply:
Hmmm
I think they were bubbling in paper

Reply:
Turns out it is direct event in paper
I will change it to direct then